vsock-bridge/src/vsock-bridge.cpp

```diff
@@ -56,20 +56,20 @@ void start_services(std::vector<service_description>& services, int numIOThreads
 
     for (int i = 0; i < numWorkers; i++)
     {
-        WorkerThread* t = new WorkerThread(
+        auto t = std::make_unique<WorkerThread>(
             /*init:*/ []() { 
                 BufferManager::arena->init(512, 2000); 
```

Comment:
Probably should've made it configurable for identity/map scenarios. Out of the scope though.